pkg/query/classes.go

```diff
@@ -0,0 +1,58 @@
+package query
+
+import (
+	"strings"
+
+	sitter "github.com/smacker/go-tree-sitter"
+)
+
+const methodsAndFields = `
+(class_definition
+  name: (identifier) @name
+  body: (block ([
+    (expression_statement (assignment)) @field
+    (function_definition) @method
+    (_)
+  ])*)
+)
+`
+
+type Class struct {
+	Name    string
+	Methods []Signature
+	Fields  []Symbol
+	Members []Symbol
+}
+
+func Classes(doc DocumentContent, node *sitter.Node) []Class {
```

Comment:
I'd probably add a comment here that starlark doesn't actually support classes but we represent builtin types + methods as python classes